### ISSUE TITLE
Mission: The Murketh and Shaadraz Gateways (BB #993)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#993]
This issue was migrated from bitbucket.
**Original Reporter:** AlphafoxBHN
**Original Date:** 07.06.2015 21:21:32 GMT+0000
**Original Priority:** major
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/993

<hr>

The quest 
Alliance:
http://www.wowhead.com/quest=10146/mission-the-murketh-and-shaadraz-gateways

and 

Horde:
http://www.wowhead.com/quest=10129/mission-gateways-murketh-and-shaadraz

Dont work.
If you drop the bomb on the Gates they dont complete the quest.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** PadreWoW
**Original Date:** 08.06.2015 06:09:20 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/993/#comment-18647883

<hr>

easy to fix (through DB)

report is not necessary

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zurr
**Original Date:** 08.06.2015 07:52:52 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/993/#comment-18649566

<hr>

any confirmed bug report 
is useful for community

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 08.06.2015 19:47:44 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/993/#comment-18669536

<hr>

@PadreWoW

> easy to fix (through DB)
> 
> report is not necessary

How can you be so... selfish?  
I could say every commit I or LordUsagi pushed to the repo was easy to fix, but why do we care to share it with world then?

Reporting, communicating, explaining, and helping each other are essential pillars.  
Without them there would have been no community nor project at all.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** PadreWoW
**Original Date:** 09.06.2015 06:05:18 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/993/#comment-18677758

<hr>

ohhhh so difficult

```
#!c++
DELETE FROM spell_script_target WHERE entry = '33655';
INSERT INTO spell_script_target VALUES (33655, 1, 19291),(33655, 1, 19292);
UPDATE `world`.`creature_ai_scripts` SET `event_flags`='1' WHERE `id`='1929101';
UPDATE `world`.`creature_ai_scripts` SET `event_flags`='1' WHERE `id`='1929201';

```

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** desteny
**Original Date:** 09.06.2015 17:20:20 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/993/#comment-18696574

<hr>

> How can you be so... selfish?
> I could say every commit I or LordUsagi pushed to the repo was easy to fix, but > why do we care to share it with world then?
> Reporting, communicating, explaining, and helping each other are essential  pillars.
> Without them there would have been no community nor project at all.

thats true, but PadreWoW was like this already years ago on the old oregoncore forum. i don't want start any offensive, but he asked for help on a lot of things (easy as well), but if he fixed something he acted like this.

nice to see... some people never change ;)

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** AlphafoxBHN
**Original Date:** 09.06.2015 18:16:40 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/993/#comment-18697814

<hr>

It dosnt work!

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** PadreWoW
**Original Date:** 10.06.2015 07:12:22 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/993/#comment-18715198

<hr>

It works on my server

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** AlphafoxBHN
**Original Date:** 10.06.2015 18:05:28 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/993/#comment-18731480

<hr>

and why not on my ?

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 06.07.2015 20:32:35 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/993/#comment-19855228

<hr>

[DB/Updates] Misc World DB Updates.

```
         Fixes #993, #1034, #1027.
```

→ <<cset 4bea6bce18b6>>
